CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+Contributing
```

Comment:
@shalithasuranga 
Adding `CONTRIBUTING.md` has to be not a part of this PR since it's not relevant or a dependency to the PR.

Reply:
Hi..
@dinukadesilva
Thanks for pointing out the issue. I removed CONTRIBUTING.md from this PR